Guests/Properties/launchSettings.json

```diff
@@ -21,7 +21,7 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "Guests",
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "applicationUrl": "https://localhost:5001;",
       "environmentVariables": {
```

Comment:
Shouldn't they all be variables? Our deployed application URL isn't `https://localhost:5001`

Reply:
I will update this when we know that the Azure is good so I don't do a lot of calls to it and costing money while still in the development stage

Reply:
If it's on a variable it shouldn't matter where it's hosted since that info will be part of the environment.